pipeline/ami_script.py

```diff
@@ -9,8 +9,8 @@
 import boto3
 from botocore.exceptions import ClientError
 
-from .script_helpers import set_default_region
-from .configuration_getters import get_configs_folder, get_aws_object_names
```

Comment:
These relative path imports didn't work for me. Not sure if it matters, but I was not using a virtual environment or pipenv. Removed them and all was well.